submit-api/src/submit_api/models/submission.py

```diff
@@ -46,6 +46,7 @@ class Submission(BaseModel):
     major_version = Column(db.Integer, nullable=False, default=1)
     minor_version = Column(db.Integer, nullable=False, default=1)
     active = Column(db.Boolean, nullable=False, default=True)
+    deleted = Column(db.Boolean, nullable=False, default=False)
```

Comment:
is_deleted might be better..but not too sure if there is a standard
But do you think if there is a scope for including this in the status somehow? like there are two flags active,deleted and status which feels like does the same thing